policyengine_us/parameters/gov/contrib/harris/rent_relief_act/rent_relief_credit/safmr_increase.yaml

```
@@ -0,0 +1,11 @@
+description: The small area fair market rent (SAFMR) is increased by this amount if used for the Housing Choice Voucher Program under the rent relief credit.
```

Fix the code to align. Easiest to subtract this amount from gross income before mapping to thresholds.
```suggestion
description: Kamala Harris proposed raising the Rent Relief Act thresholds by this amounts for residents of areas where HUD uses the small area fair market rent for the Housing Choice Voucher Program.
```

---

This would generally better align with our general code standards, I wonder if we would run into questions regarding ownership if it is e.g. a co-sponsered proposal and we attribute all accountability to one person / org.

---

Good point, though I think where ownership is clear we can adopt this new standard.

---

Agreed